config/site_settings.yml

```diff
@@ -2346,13 +2346,6 @@ developer:
   bypass_wizard_check:
     default: false
     hidden: true
-  logging_provider:
```

Comment:
Might need a migration to delete from site_settings too

Reply:
O good catch 👍